keras/backend/jax/distribution_lib.py

```diff
@@ -84,6 +82,8 @@ def distribute_tensor(tensor, layout):
     Returns:
         Distributed value.
     """
+    from keras.utils import jax_utils
```

Comment:
Was that for a circular dependency?

Reply:
Yes. It turns out that when you `import keras`, `jax_utils` was immediately and always imported.